.travis.yml

```diff
@@ -24,7 +24,8 @@ env:
 install:
   - npm install yarn -g
   - yarn install
-  - if [ ! -h "$HOME/.yarn-cache/.link/webpack" ]; then yarn link; fi
+  # - if [ ! -h "$HOME/.yarn-cache/.link/webpack" ]; then yarn link; fi
+  - yarn link || yarn link webpack
```

Comment:
Why do you fallback to `yarn link webpack` here? One line after this it also does that. If you want to ignore the error `yarn link` gives if you run it twice, it's cleaner to do `yarn link || true`